### ISSUE TITLE
unban more than 1 ip at once

### ISSUE THREAD

**Issue by voicenoise** [fail2ban/fail2ban#2349]
hello

i try:
`fail2ban-client set nginx-x00 unbanip xxx.xxx.xxx.xx1, xxx.xxx.xxx.xx2, xxx.xxx.xxx.xx3` e.g.

but its not possible only the first ip are unbanned. is there another option to unban more than one IP adress at once ?

thanks

**Comment by sebres** [comment]
At the moment is is impossible **for single jail** (only one IP at a time).
But if your fail2ban is >= 0.10, you can do it **for all jails** or all IPs for single jail:
```bash
# unban IPs in all jails:
fail2ban-client unban <ip1> ... <ipN>
# or unban all IPs across all jails:
fail2ban-client unban --all
# or unban all IPs for single jail:
fail2ban-client reload --unban <JAIL>
```
It is relative simply to implement (so I'll try to provide solution for that).

**Comment by sebres** [comment]
implemented in #2351

**Comment by voicenoise** [comment]
cool thanks for the info i try it soon (y)
